.github/workflows/docker.yml

```diff
@@ -52,6 +56,18 @@ jobs:
       - name: Run brew test-bot --only-setup
         run: docker run --rm brew brew test-bot --only-setup
 
+      - name: Generate image digest
+        id: digest
+        run: echo "digest=$(docker inspect --format='{{index .RepoDigests 0}}' brew)" >> "$GITHUB_OUTPUT"
```

Comment:
Based on the CI failures, this step might be failing -- the `attest-build-provenance` below is failing with `subject-digest` missing, suggesting that it's being set to an empty string here.

(Maybe because of the `docker run --rm` above? I'm not so good with Docker 🙂)

Reply:
```suggestion
      - name: Generate image digest
        id: digest
        run: echo "digest=$(docker image inspect --format='{{.Digest}}' brew)" >> "$GITHUB_OUTPUT"
```